docs/resources/on_call_escalation_policy.md

```diff
@@ -83,6 +83,6 @@ Import is supported using the following syntax:
 The [`terraform import` command](https://developer.hashicorp.com/terraform/cli/commands/import) can be used, for example:
 
 ```shell
-# Import an existing on_call_schedule
-terraform import datadog_on_call_schedule.test "b03a07d5-49da-43e9-83b4-5d84969b588b"
+# Import an existing on_call_escalation_policy
+terraform import datadog_on_call_escalation_policy.test "b03a07d5-49da-43e9-83b4-5d84969b588b"
```

Comment:
```
terraform import datadog_on_call_escalation_policy.policy_test "b03a07d5-49da-43e9-83b4-5d84969b588b"
```
Still no good since on the example the key is `policy_test`